.github/workflows/release.yml

```diff
@@ -24,6 +24,51 @@ jobs:
       - id: changelog-version
         run: echo "version=$(echo "${{ inputs.versionNumber }}" | cut -c 2-)" >> "$GITHUB_OUTPUT"
 
+  releasedVersion:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.remove-leading-v.outputs.version }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-ecosystem/action-get-latest-tag@v1
+        id: get-latest-tag
+        with:
+          semver_only: true
+
+      - name: Remove leading v from tag
+        id: remove-leading-v
+        run: |
+          VERSION='${{ steps.get-latest-tag.outputs.tag }}'
+          echo "VERSION=${VERSION:1}" >> $GITHUB_OUTPUT
+
```

Comment:
This workflow already extracts the semantic version via the `changelog-version` job (which could potentially be renamed for clarity). I'm also curious why `cdktf-registry-docs` needs the version when being directly provided a binary?

```suggestion
```

Reply:
I looked into it and it turns out I don't actually need the version number, it was based on a misconception. I will change the tool to not require it anymore :)